src/modules/cloudflare/turnstile/turnstile.guard.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  BadRequestException,
+  CanActivate,
+  ExecutionContext,
+  Inject,
+  Injectable,
+} from '@nestjs/common';
+
+import { TurnstileOptions } from './interfaces/turnstile-options.interface';
+import { TurnstileService } from './turnstile.service';
+
+export enum Messages {
```

Comment:
This is a personal guess, we don't need this `enum`, it is being initialized too close to where it is used, this adds unnecessary complexity in a small block of code...

Reply:
Maybe make sense move to a interface file

Reply:
Yeah, I'm agree about your point.